BWWalkthrough/BWWalkthroughViewController.swift

```diff
@@ -287,7 +287,7 @@ extension UIView {
         Removes only constraints pertaining to the width or height of this view.
      */
     func removeHeightWidthConstraints() {
-        for constraint in constraints() as! [NSLayoutConstraint] {
+        for constraint in constraints {
```

Comment:
Otherwise I didn't have trouble understanding the new changes, but this one puzzled me. Is `constraints` a property where it previously was not, or what makes this change possible / necessary @Infinity-James ?